server/status.go

```diff
@@ -92,6 +92,10 @@ const (
 	// healthEndpoint is a shortcut for local details, intended for use by
 	// monitoring processes to verify that the server is up.
 	healthEndpoint = "/health"
+
+	// prometheusEndpoint is the prometheus metrics endpoint, not to be
+	// confused with the built-in cockroach metrics.
+	prometheusEndpoint = "/metrics"
```

Comment:
I've used `/metrics` for now only because it's the default, but it's configurable.
It may make more sense to have it as a variation of our own metrics endpoint, or at least at a similar path.